config/config.ini

```diff
@@ -2397,6 +2388,9 @@ name = Vinay Sajip
 [http://plumberjack.blogspot.com/feeds/posts/default]
 name = Vinay Sajip (Logging)
 
+[https://www.vinta.com.br/python/feed/]
+name = Vinta Software
+
```

Comment:
Please update the PR to include *only* this part of the patch:  perturbing others' locations within the file makes the PR too hard to review.

Reply:
@tseaver Hi! How are you? The docs said I had to run that sorting script before adding changes. When I ran that script this other stuff got sorted too. Should I not run the sorting script, add Vinta's feed in the correct spot manually and update the PR? Thanks!

Reply:
@amandasavluchinske Ah, I see -- those notes are really for the maintainers, rather than folks adding / updating their own feeds.  I will compare your diff with the one I get locally from running the script, and merge if everything matches.